examples/creating_a_turtle.rs

```diff
@@ -0,0 +1,307 @@
+extern crate turtle;
+
+use turtle::Turtle;
+
+const SIZE: f64 = 3.0;
+
+fn main() {
+    let mut turtle = Turtle::new();
+    turtle.set_speed(8);
+
+    turtle.pen_up();
+    turtle.set_x(-280.0);
+    turtle.set_y(-90.0);
+
+    // Panzer
+    {
```

Comment:
Could you split up these sections into multiple functions? Anytime you indented things into a separate expression using `{ ... }` you can probably just move that code into a named function.

With really long examples like this, it's a great idea to split things up into more manageable chunks using functions and then document each of the different parts of the code (kind of like you did with the tree example you contributed). The [rust logo example](https://github.com/sunjay/turtle/blob/master/examples/rust.rs) is a pretty good example of splitting things up into many functions. It could definitely use more comments though, so please do better at that than I did. :smile: 

There is a lot of code here, so we want to make sure it is manageable and readable for someone who might not be able to see exactly what is going on right when they look at it. Any repeated code should definitely be moved into a function and reused to make this example shorter.